### PR TITLE
Exercise: Does my coding partner follow JS best practices?

### PR DESCRIPTION
**Pull request by Mukumbuta**
### Preparatory procedure
- Created a new empty repository.
- Created a feature branch called P2P-Review-Code
- Copied code from my "To Do List" project and commited it as one big chunk of code in the feature branch.
- Invited my coding partner to collaborate and review in [GitHub](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review).
- Shared the pull request link with my coding partner.

### CODE REVIEW

**Comment by AbigiyaTY** [comment]
Hey @Mukumbuta 
Good job so far 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Highlights

- Good PR description 
- No linter errors.

# Required Changes
 Please, consider fixing the following requirement:
- In the style.css file lines 104 and 108 have the same value so I believe it is good to bring them together also for the font family you can assign it to the body so there wouldn't be code redundancy.
- In the js file, instead of using for loop it is good practice to use map or forEach for readability and clean code [KISS => Keep It Simple, Smart](https://medium.com/infancyit/kiss-keep-it-simple-smart-javascript-habits-b019ce14b5ec)
- Also, I suggest you remove the comments, to make your code clear and professional. Your code is enough understandable and talks by itself about what is happening 👍
- I see a live demo so I think it is better to have a live demo on your readme file.

# Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

    N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @AbigiyaTY

**Comment by Mukumbuta** [comment]
Hi @AbigiyaTY 
Thank you for the very professional and detailed code review. 
However, I seek clarity on the following issues: 
- [ ] I used a 'traditional' for-loop just to generate numbers to append to the key that would hold the value of each checkbox status. Am not using the for loop to traverse throught the Array `getTodos`. So am not sure if I was still supposed to use a forEach loop. Please clarify.
- [ ] The values on lines 104 and 108 are not the same and am calling the rules on two different classes (The `image ` class has `display: none` while the `remove ` class has `display: flex`). Are you sure you got the line numbers correctly?
- [ ] Lastly, am setting the` font-family: cursive` rule on selected elements. As such setting the rule in the body will enforce it on all elements. I don't think that's what I want. 
Please clarify asap so I can quickly jump into enforcing the changes you've required.

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  display: none;
+}
+
+.image {
+  display: flex;
+}
+
+.remove {
+  display: none;
+}
```
Comment (AbigiyaTY) [inline src/style.css]: Sorry for the confusion this is the part I wrote about.

Comment (Mukumbuta) [reply src/style.css]: Alright, got it. Thank you. I will definitely implement it.

**Comment by Mukumbuta** [comment]
Hi @AbigiyaTY 
I have implemented all the changes you required in your previous code review. If you get some time, please review my new changes. Thank you, in anticipation.

**Comment by AbigiyaTY** [comment]
Hello @Mukumbuta  ✋,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉
Highlights 💪

      - Project works fine
      - clean and simple UI
      - no linter errors

Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@AbigiyaTY ](https://github.com/AbigiyaTY) in your question so I can receive the notification.

**Comment by Mukumbuta** [comment]
Thank you so much, @AbigiyaTY , for the code review and approval.

**Pull request merged**